style-guide/readme.md

```diff
@@ -0,0 +1,47 @@
+> **Goal of this guide:** to help you write to fit a specific context, purpose, or audience.
+
+# How to refine your writing style
+
+## Recognise what style is and what it's building blocks are
```

Comment:
The first heading is a title. Do you still think there needs to be something in between?